preload/preload-resource-match.https.sub.html

```diff
@@ -0,0 +1,156 @@
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="/preload/resources/preload_helper.js"></script>
+<script>
+
+const crossOriginHost = 'https://{{host}}:{{ports[https][0]}}'
+
+function createEchoURL(text, type) {
+    return `/preload/resources/echo-with-cors.py?type=${
+        encodeURIComponent(type)}&content=${
+        encodeURIComponent(text)}`
+}
+const urls = {
+    image: createEchoURL('<svg xmlns="http://www.w3.org/2000/svg" width="2" height="2" />', 'image/svg+xml'),
+    font: '/preload/resources/font.ttf?x',
+    text: createEchoURL('hello', 'text/plain'),
+    script: createEchoURL('function dummy() { }', 'application/javascript'),
+    style: createEchoURL('.cls { }', 'text/css'),
+}
+
+const resourceTypes = {
+    image: {url: urls.image, as: 'image'},
+    font: {url: urls.font, as: 'font', config: 'anonymous'},
+    backgroundImage: {url: urls.image, as: 'image', config: 'no-cors'},
+    fetch: {url: urls.text, as: 'fetch'},
+    script: {url: urls.script, as: 'script'},
+    module: {url: urls.script, as: 'script'},
+    style: {url: urls.style, as: 'style'}
+}
+
+const configs = {
+    'same-origin': {crossOrigin: false, attributes: {}},
```

Comment:
Can you add a comment saying what each of the config values does?

Reply:
Done